doc/ug/install_instructions/index.md

```diff
@@ -77,10 +77,9 @@ To build device software you need a baremetal RV32IMC compiler toolchain.
 You can either build your own or use a prebuilt one.
 We recommend installing the toolchain to `/tools/riscv`.
 
-#### Option 1 (recommended): Use the lowRISC-provided prebuilt GCC toolchain
+#### Option 1 (recommended): Use the lowRISC-provided prebuilt toolchain
 
-lowRISC provides a prebuilt GCC toolchain for the OpenTitan project.
-Download the file starting with `lowrisc-toolchain-gcc-rv32imc-` from [GitHub releases](https://github.com/lowRISC/lowrisc-toolchains/releases/latest) and unpack it to `/tools/riscv`.
+lowRISC provides a prebuilt toolchain for the OpenTitan project. This toolchain contains both GCC and Clang, targeting RISC-V. By default the device software is built with Clang. We recommend using the `util/get-toolchain.py` tool to download and install the latest toolchain. This tool will automatically adjust the toolchain configuration if you override the installation directory. Alternatively, manually download the file starting with `lowrisc-toolchain-rv32imc-` from [GitHub releases](https://github.com/lowRISC/lowrisc-toolchains/releases/latest) and unpack it to `/tools/riscv`.
```

Comment:
One sentence per line, please.

You also might want to update the introduction to this paragraph, which says "You can either build your own or use a prebuilt one.", to be stronger about the recommendation to use the pre-built one.